requirements.txt

```diff
@@ -6,3 +6,4 @@ requests
 typing-extensions
 # Newer version does not work with default OpenSSL version on jammy.
 cryptography <= 38.0.4
+pydantic>=1,<2
```

Comment:
Would you mind pinning this?

Reply:
I have pinned it more specifically, but not to an exact version. I am a bit hesitant, as the other packages in this file are also not specifically pinned, and it seems that we also need to agree on pinning as a team, taking into account @jdkandersson 's blog post https://jdkandersson.com/2023/09/18/to-pin-or-not-to-pin-the-version/ . cc @yhaliaw